src/strategies/arbitrum/uniswapv3/strategy-univ3-eth-usdc-lp.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "../../strategy-univ3-rebalance.sol";
+
+contract StrategyUsdcEthUniV3Arbi is StrategyRebalanceUniV3 {
+    address public usdc_eth_pool = 0xC31E54c7a869B9FcBEcc14363CF510d1c41fa443;
+    address[] public rewardTokens = [address(token0), address(token1)];
+
+    constructor(
+        int24 _tickRangeMultiplier,
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategyRebalanceUniV3(usdc_eth_pool, _tickRangeMultiplier, _governance, _strategist, _controller, _timelock)
+    {
+        univ3_staker = 0x1f98407aaB862CdDeF78Ed252D6f557aA5b0f00d;
+
+        key = IUniswapV3Staker.IncentiveKey({
```

Comment:
I understand that the Uni V3 staking stuff are artifacts from the Ribbon code, but a lot of it should be cleaned out for the USDC-ETH implementation (which presumably doesn't have any staking rewards)

Reply:
Agreed, I'm thinking to make a seperate abstract class for a staker, and an abstract class without staking.